servicetalk-concurrent-internal/src/main/java/io/servicetalk/concurrent/internal/SignalOffloader.java

```diff
@@ -28,7 +28,7 @@
  * href="https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.3/README.md#glossary">signals</a> to and
  * from any asynchronous source.
  *
- * <h2>Caution</h2>
+ * <strong>Caution</strong>
```

Comment:
@bondolo - let me know when https://github.com/apple/servicetalk/pull/1639 is merged, and I will rebase this PR and remove these changes.